ui/src/app/views/concept-homepage/component.spec.ts

```diff
@@ -224,4 +219,30 @@ describe('ConceptHomepageComponent', () => {
     // After select add the number of selected concepts
     expect(buttonText).toBe('Add (1) to set');
   }));
+
+  it('should clear selected count after adding', fakeAsync(() => {
+    const de = fixture.debugElement;
+
+    simulateClick(fixture, de.query(By.css('.standard-concepts-checkbox')).children[0]);
+    simulateInput(fixture, de.query(By.css('#concept-search-input')), 'test');
+    simulateClick(fixture, de.query(By.css('.btn-search')));
+    updateAndTick(fixture);
+    const dataRow = de.queryAll(By.css('.concept-row'));
+    const checkBox = dataRow[0].queryAll(By.css('.datagrid-select'))[0]
+        .query(By.css('.checkbox')).children;
+    simulateClick(fixture, checkBox[0]);
+    updateAndTick(fixture);
+
+    simulateClick(fixture, de.query(By.css('.sliding-button')));
+    updateAndTick(fixture);
+
```

Comment:
Nit: I understand the  button "btn-primary" is for the add concept set to concept model with the option of 'existing concept set'. Do you think it will be informative to add comment here. 
My only concern is since it is indirectly relaying on the stubConceptSets from ConceptSetsServiceStub and in case anyone make any changes to it or change domain (i know they shouldnt) it will not be very easy to debug why this test failed because of that.

Reply:
Good catch, thanks. I didn't actually think about that when I was writing the test.

I decided to just create a new concept set instead to sidestep this issue.